torch/fx/experimental/sym_node.py

```diff
@@ -240,6 +240,15 @@ def __str__(self):
         return self.str()
 
     def __repr__(self):
+        rep = [
+            f"SymNode({self.expr}, shape_env={self.shape_env}, pytype={self.pytype}",
+            *(f"hint={self._hint}" for _ in [None] if self._hint is not None),
+            *(f"constant={self.constant}" for _ in [None] if self.constant is not None),
+        ]
+        return ", ".join(rep) + ")"
```

Comment:
Let's make this a bit easier to read - something like:
```
rep = [str(self.expr), f"shape_env={self.shape_env}", f"pytype={self.pytype}"]
if self._hint is not None:
    rep.append(f"hint={self._hint}")
if self.constant is not None:
    rep.append(f"constant={self.constant}")
return "SymNode(" + ", ".join(rep) + ")"
```

Reply:
Sure! I thought it was more elegant, but it is indeed harder to read. 
I also realized that I forgot to add the attribute `fx_node`. Seeing that you didn't flag it, should I add it or is it unnecessary?